.github/workflows/listgen.yml

```diff
@@ -0,0 +1,115 @@
+name: Generate stdlib lists
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 0"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Are Sundays the best day for this action to fire?

Reply:
92e5670..d624d81